clients/clearly-defined/src/main/kotlin/Enums.kt

```diff
@@ -113,27 +87,20 @@ enum class Provider {
  */
 @Serializable
 enum class Nature {
-    @SerialName("license")
-    LICENSE,
-    @SerialName("notice")
-    NOTICE
+    @SerialName("license") LICENSE,
+    @SerialName("notice") NOTICE
 }
 
 /**
  * See https://github.com/clearlydefined/website/blob/43ec5e3/src/components/ContributePrompt.js#L78-L82.
  */
 @Serializable
 enum class ContributionType {
-    @SerialName("Missing")
-    MISSING,
-    @SerialName("Incorrect")
-    INCORRECT,
-    @SerialName("Incomplete")
-    INCOMPLETE,
-    @SerialName("Ambiguous")
-    AMBIGUOUS,
-    @SerialName("Other")
-    OTHER
+    @SerialName("Missing") MISSING,
```

Comment:
Not sure if we agreed already on having these on a single line. If so, please just ignore this comment:

The approach seems to work as long as there are only 1 annotation per value / property. As soon as one needs to add a second annotation, it will become ugly. Would we but 2,3,4 annotations on a single line then? How would it look then?